src/api/app/controllers/webui/groups_controller.rb

```diff
@@ -4,20 +4,20 @@ class Webui::GroupsController < Webui::WebuiController
   after_action :verify_authorized, except: [:show, :autocomplete]
 
   def index
-    authorize Group, :index?
+    authorize Group.new, :index?
```

Comment:
This is a bit weird, we're not going to create groups in the `index` view. Is this change really needed?

Reply:
```
[4] pry(main)> Group.new.class.name
=> "Group"
[5] pry(main)> Group.class.name
=> "Class"
```